test/WaveEquationCubedSphereTests.jl

```diff
@@ -142,6 +151,7 @@ function solve_wave_equation_ssrk2(
       pe  = Vector{Float64}(undef,N)
       kin_to_pot = Vector{Float64}(undef,N)
       pot_to_kin = Vector{Float64}(undef,N)
+      mass = Vector{Float64}(undef,N)
```

Comment:
This memory allocation is ok. We allocate before the time integration loop. Not at each iteration, like in the previous function.

Reply:
Besides, as we know the size of the vectors beforehand, we can allocate to given size, instead of initializing to the empty vector and then pushing entries to the end (which again, requires frequent memory allocation)